source/includes/fundamentals/code-snippets/Delete.java

```diff
@@ -52,35 +52,41 @@ public static void main(String [] args){
     }
 
     private void deleteManyExample(){
+        // Deletes all documents in the collection that contain a "qty" value of 0
         // begin deleteManyExample
         Bson filter = Filters.eq("qty", 0);
         collection.deleteMany(filter);
         // end deleteManyExample
     }
 
     private void findOneAndDeleteExample(){
+        // Deletes the first document with a "color" value of "purple" and prints the deleted document
         // begin findOneAndDeleteExample
         Bson filter = Filters.eq("color", purple);
         System.out.println(collection.findOneAndDelete(filter).toJson());
         // end findOneAndDeleteExample
     }
 
     private void findOneAndDeleteNullExample(){
+        // Deletes the first document with a "qty" value of 1 and prints the deleted document
         // begin findOneAndDeleteNullExample
         Bson filter = Filters.eq("qty", 1);
         System.out.println(collection.findOneAndDelete(filter));
         // end findOneAndDeleteNullExample
     }
 
     private void deleteOneExample(){
+        // Deletes the first document with a "color" value of "yellow"
         // begin deleteOneExample
         Bson filter = Filters.eq("color", "yellow");
         collection.deleteOne(filter);
         // end deleteOneExample
     }
     private void preview(boolean drop){
+        // Prints the results of a find operation as JSON
```

Comment:
Suggestion:
I think it could be good to keep this consistent with other comments on [similar code](https://github.com/mongodb/docs-java/pull/473/files#diff-7e0bf74db4086e8b59e29baf324c16c5e80612136e85de47665dca4002236c06R92).